frontend/src/components/forms/InviteEmployeesModal.tsx

```diff
@@ -0,0 +1,314 @@
+      setInvitedLastNameError(false);
+    } else {
+      setInvitedLastNameError(true);
+    }
+  };
+
+  const handleEmailChange = (e: { target: { value: unknown } }) => {
+    const inputValue: string = e.target.value as string;
+    setInvitedEmail(inputValue);
+    if (inputValue.endsWith("@supportivehousing.com")) {
+      setInvitedEmailError(false);
+    } else {
+      setInvitedEmailError(true);
+    }
+  };
+
+  const handleAdminChange = (inputValue: string) => {
+    setinvitedAdminStatus(inputValue);
+    if (inputValue !== "") {
+      setInvitedAdminStatusError(false);
+    } else {
+      setInvitedAdminStatusError(true);
+    }
+  };
+
+  const handleOpen = () => {
+    setIsOpen(true);
+    setInvitedEmail("");
+    setInvitedFirstName("");
+    setInvitedLastName("");
+    setinvitedAdminStatus("");
+    setIsTwoFactorAuthenticated(false);
+
+    setInvitedEmailError(false);
+    setInvitedFirstNameError(false);
+    setInvitedLastNameError(false);
+    setInvitedAdminStatusError(false);
+  };
+
+  const handleClose = () => {
+    setIsOpen(false);
+  };
+
+  const onInviteEmployee = async () => {
+    if (
+      invitedEmail !== "" &&
+      invitedFirstName !== "" &&
+      invitedLastName !== "" &&
+      invitedAdminStatus !== ""
+    ) {
+      let hasInvitedUser = false;
+      if (invitedAdminStatus === "1") {
+        hasInvitedUser = await commonApiClient.inviteUser(
+          invitedEmail,
+          RoleOptions[1],
+          invitedFirstName,
+          invitedLastName,
+        );
+      } else if (invitedAdminStatus === "2" && isTwoFactorAuthenticated) {
+        hasInvitedUser = await commonApiClient.inviteUser(
+          invitedEmail,
+          RoleOptions[0],
+          invitedFirstName,
+          invitedLastName,
+        );
+      } else if (invitedAdminStatus === "2" && !isTwoFactorAuthenticated) {
+        hasInvitedUser = await commonApiClient.inviteUser(
+          invitedEmail,
+          RoleOptions[2],
+          invitedFirstName,
+          invitedLastName,
+        );
+      }
+      // TO DO: add toast for error and success
+      if (!hasInvitedUser) {
+        newToast("Error inviting employee", INVITE_EMPLOYEE_ERROR, "error");
+      } else {
+        newToast(
+          "Invite sent",
+          `Your invite has been sent to ${invitedFirstName} ${invitedLastName}`,
+          "success",
+        );
+        handleClose();
+      }
+    }
+  };
+
+  const handleSubmit = () => {
+    setInvitedEmailError(invitedEmail === "");
+    setInvitedFirstNameError(invitedFirstName === "");
+    setInvitedLastNameError(invitedLastName === "");
+    setInvitedAdminStatusError(invitedAdminStatus === "");
+
+    if (
+      !invitedEmailError &&
+      !invitedFirstNameError &&
+      !invitedLastNameError &&
+      !invitedAdminStatusError
+    ) {
+      onInviteEmployee();
+    }
+  };
+
+  return (
+    <>
+      <Button size="lg" variant="solid" colorScheme="pink" onClick={handleOpen}>
```

Comment:
pink button goes crazy 🪅 

I would change this to what we would use in the app, you can remove the styling and just use the primary variant

```suggestion
      <Button variant="primary" onClick={handleOpen}>
 ```
 
 and also change the button text so it has an add icon and the text is something like `Invite Employee`